protocol/development-workflow/README.md

```diff
@@ -66,6 +66,8 @@ Merge the release branch into the QA branch.
 
     git merge <release-name>
 
+Copy the QA steps from the Pivotal Tracker story into the PR description, and put the story ID in the PR title.
```

Comment:
I also like to add a link to PT in the PR description, as it's easier to click that than find a story by its ID.

I do this on all PRs I submit that have a story connected to them -- so both dev and QA. I think including this step (adding PT link) both here and under "Developing for release" is a good idea.